### PR TITLE
Feature/align checkbox

### PR DESCRIPTION
**Pull request by engordon**

### CODE REVIEW

File: scss/common/_animation.scss [hunk 1]
```diff
@@ -70,6 +70,7 @@ Enable to expend a folded block element
 .sdc-ripple-click__animated::before{
   display: inline-block;
   position:absolute;
+  top:0;
   content: '';
   animation: ripple-animation .3s ease-out; 
   background-color: #009fdb;
```
Comment (islavi) [inline scss/common/_animation.scss]: Please replace the background from SCSS

File: scss/common/_animation.scss [hunk 2]
```diff
@@ -70,6 +70,7 @@ Enable to expend a folded block element
 .sdc-ripple-click__animated::before{
   display: inline-block;
   position:absolute;
+  top:0;
   content: '';
```
Comment (islavi) [inline scss/common/_animation.scss:74]: Do not use specific width and height (14px)

File: scss/common/_animation.scss (hunk 1 above)
Comment (ArielDKenan) [inline scss/common/_animation.scss]: Why is z-index being used here? If the animation looks good without it, or if you can get the behavior right using 'order' instead, then please remove z-index

Comment (engordon) [reply scss/common/_animation.scss]: Agreed, there's no need for z-index here ('order' isnt related though).

File: scss/common/_animation.scss (hunk 2 above)
Comment (islavi) [reply scss/common/_animation.scss:74]: Keep 14px for all radio / checkbox / close buttons ("X")

**Review by islavi — APPROVED** [review]

**Pull request merged**